apps/ukvi-complaints/fields/index.js

```diff
@@ -320,21 +320,21 @@ module.exports = {
   'vac-country': {
     mixin: 'select',
     options: [{ label: ' ', value: '' }].concat(require('hof-util-countries')()),
-    validate: 'required',
+    validate: ['required', { type: 'maxlength', arguments: 100 }],
```

Comment:
is this because they have a maxlength on the DECS side?

Reply:
Yeah, they have set some max lengths on the json schema we are validating against. We want to prevent the user from exceeding it here rather than allow them to complete the form and then throw an error when it fails during schema validation.